iceoryx_posh/source/mepoo/chunk_header.cpp

```diff
@@ -23,12 +23,12 @@ namespace iox
 namespace mepoo
 {
 ChunkHeader::ChunkHeader(const uint32_t chunkSize, const ChunkSettings& chunkSettings) noexcept
+    : m_chunkSize(chunkSize)
+    , m_userPayloadSize(chunkSettings.userPayloadSize())
 {
     static_assert(alignof(ChunkHeader) >= 8U,
```

Comment:
I think this should move in the class definition of the chunk header since it is a class requirement.

Reply:
not possible, since `ChunkHeader` is not complete at that point and `alignof` needs a complete type

Reply:
@MatthiasKillat the calculations rely on the alignment of the `ChunkHeader` to be at least 8. Therefore it's also at the place where the calculation happens